plugins/modules/aci_l3out_floating_svi.py

```diff
@@ -0,0 +1,470 @@
+  description: The original configuration from the APIC before the module has started
+  returned: info
+  type: list
+  sample:
+    [
+        {
+            "fvTenant": {
+                "attributes": {
+                    "descr": "Production",
+                    "dn": "uni/tn-production",
+                    "name": "production",
+                    "nameAlias": "",
+                    "ownerKey": "",
+                    "ownerTag": ""
+                }
+            }
+        }
+    ]
+proposed:
+  description: The assembled configuration from the user-provided parameters
+  returned: info
+  type: dict
+  sample:
+    {
+        "fvTenant": {
+            "attributes": {
+                "descr": "Production environment",
+                "name": "production"
+            }
+        }
+    }
+filter_string:
+  description: The filter string used for the request
+  returned: failure or debug
+  type: str
+  sample: ?rsp-prop-include=config-only
+method:
+  description: The HTTP method used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: POST
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec, aci_contract_dscp_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(
+        tenant=dict(type="str", aliases=["tenant_name"], required=True),
+        l3out=dict(type="str", aliases=["l3out_name"], required=True),
+        node_profile=dict(type="str", aliases=["node_profile_name", "logical_node"], required=True),
+        interface_profile=dict(type="str", aliases=["interface_profile_name", "logical_interface"], required=True),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+        pod_id=dict(type="str"),
+        node_id=dict(type="str"),
+        address=dict(type="str", aliases=["addr", "ip_address"]),
+        link_local_address=dict(type="str"),
+        mac_address=dict(type="str"),
+        mtu=dict(type="str"),
+        ipv6_dad=dict(type="str", choices=["enabled", "disabled"]),
+        mode=dict(type="str", choices=["regular", "native", "untagged"]),
+        encap=dict(type="str"),
+        encap_scope=dict(type="str", choices=["vrf", "local"]),
+        auto_state=dict(type="str", choices=["enabled", "disabled"]),
+        external_bridge_group_profile=dict(type="str"),
+        dscp=aci_contract_dscp_spec(direction="dscp")
```

Comment:
`dscp=aci_contract_dscp_spec(direction="dscp")` -> `dscp=aci_contract_dscp_spec(direction="dscp"),` 
To check Black sanity test.

Reply:
thanks!